packages/competence/bin/static/fragments/frame-employee-import.html

```
@@ -0,0 +1,109 @@
+                    <template x-if="wasApplied()">
+                        <span class="ti-status-pill success" x-text="appliedSummary()"></span>
+                    </template>
+                </div>
+            </template>
+        </div>
+
+        <div class="competence-panel-body">
+            <p class="ti-form-hint" x-text-label="interface.employee-import.export-hint">Export from the template with File &gt; Save As &gt; CSV UTF-8. Another encoding stores names incorrectly and is refused.</p>
+
+            <div class="ti-form-row">
+                <input class="ti-input" type="file" accept=".csv,text/csv" x-bind:disabled="busy" @change="chooseFile($event)">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)